lib/ibm_cloud_sdk_core/authenticators/basic_authenticator.rb

```diff
@@ -24,7 +24,6 @@ def initialize(vars)
     def authenticate(headers)
       base64_authentication = Base64.strict_encode64("#{@username}:#{@password}")
       headers["Authorization"] = "Basic #{base64_authentication}"
-      headers
```

Comment:
If I understang Ruby's messed up syntax, removing this line simply means that the function no longer returns the updated set of headers, right?   But, that should be fine since we don't expect the function to return the headers... it simply adds a new entry to the set of headers passed in, therefore the caller sees the change in the headers map.  I think :)

Reply:
Yeah, that should be fine.